libs/oidc/provider-nestjs/src/lib/controllers/role/role.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Controller, Post, Req, Res, Get } from '@nestjs/common';
+import { RoleService } from '../../services/role/role.service';
+
+@Controller('role')
+export class RoleController {
```

Comment:
Can't help but notice here there is a problem when using @Req instead of @Body but not elsewhere. They're practically the same; do you want me to replace @Req EVERYWHERE with @Body?